addons/airway/stringtable.xml

```diff
@@ -1801,6 +1801,7 @@
             <Italian>KAT - ADV Medicale: Vie Aeree</Italian>
             <German>KAT - ADV Medical: Atemwege</German>
             <Japanese>KAM - 気道 (Airway)</Japanese>
+            <Czech>KAT - ADV Zdravotní modul: Horní dýchací cesty</Czech>
```

Comment:
imo omit modul, keep it as ADV Zdravotní, I would even go as far as `ADV Medické`. If you wanna go all in `Pokročilé medické`, would also fit, but I dislike the length of the name. Would of course apply to all subsequent translations.
![image](https://github.com/user-attachments/assets/129d1807-4bfb-4583-9fc8-a31760470aaf)